src/composables/useSpacefinderSupabase.ts

```diff
@@ -32,5 +34,56 @@ export default function useSpacefinderSupabase() {
     buildingsRealtimeChannel.subscribe();
   }
 
-  return { getBuildingsOccupancyCurrent, subscribeToBuildingsOccupancy };
+  async function getSpacesOccupancyCurrent() {
+    const { data, error, status } = await client
+      .from("spaces_latest_states")
+      .select("*");
+    const activeDevicesPerBuildingAndSpace: Record<
```

Comment:
Would this be easier to understand renaming `data` and using `reduce` to get the `activeDevicesPerBuildingAndSpace`?

```js
const { data: rows = [], error, status } = await client
      .from("spaces_latest_states")
      .select("*");

const activeDevicesPerBuildingAndSpace = rows.reduce(...
```